app/allocation/controllers/cancel.js

```diff
@@ -2,6 +2,16 @@ const FormWizardController = require('../../../common/controllers/form-wizard')
 const allocationService = require('../../../common/services/allocation')
 
 class CancelController extends FormWizardController {
+  middlewareLocals() {
+    super.middlewareLocals()
+    this.use(this.setCancelUrl)
+  }
+
+  setCancelUrl(req, res, next) {
```

Comment:
Nowt wrong with this, but I do find this really flabby.

Could we not do something like I originally did for the edit/updates? 

`cancelUrl: '/allocation/:allocationId'`

and then use pathToRegExp to compile and add the var

Reply:
(Not a suggestion for this PR)

Reply:
👍